CONTRIBUTING.md

```diff
@@ -56,3 +56,8 @@ Pull requests are the best way to propose changes to the notebooks repository:
 ### Testing your PR locally
 
 - Test the changes locally, by manually running the `$make jupyter-${NOTEBOOK_NAME}-ubi8-python-3.8` from the terminal. This definitely helps in that initial phase.
+
+### Review and Merge process
+
+- Once the PR is submitted, you can either select particular reviewers or leave the bot to select reviewers automatically.
+- To make PR merged, it has to get 2 `/lgtm` comments first from the repository [approvals/reviewers](/OWNERS). Then an `/approve` comment must be added by someone with approval rights. If the author of the PR has approval rights, it's preferred that he's the one who performs the merge action.
```

Comment:
**AI says**:

Sure, here is a gender-neutral and fluent version of the document:

```suggestion
### Review and Merge Process

- Once the PR is submitted, you can either select specific reviewers or let the bot to select reviewers [automatically](https://prow.ci.openshift.org/plugins?repo=opendatahub-io%2Fnotebooks).
- For the PR to be merged, it must receive 2 `/lgtm` comments from the repository [approvals/reviewers](/OWNERS). Following that, an `/approve` comment must be added by someone with approval rights. If the author of the PR has approval rights, it is preferred that they perform the merge action.
```

Reply:
thank you, I updated the text :slightly_smiling_face: